### DISCUSSION TITLE
Microbial network plotting using igraph and Gephi

### DISCUSSION THREAD

**Discussion by khasheee** [gephi/gephi#2344] [Q&A]
I want to create a network from correlation matrix of soil microbial data. I have seen in several published papers, most of them used igraph and hmisc packages for calculation of corelation matrix (Spearman’s correlation coefficient(ρ)>0.6 for +ve correlation and <0.6 as negative, and statistical significance (P<0.01)) and plotting network. Then, all of them used Gephi for network visualization.

The data I have is bacterial relative abundance at OTU level at different classification levels (from phylum to genus).

Example:

(Column 1)Bacteria|Actinobacteria|Actinobacteria|Acidimicrobiales|Iamiaceae (Rest of the columns are the relative abundance in different treatments).

For the network I want to plot, the nodes would be genera, the size of node should be based on number of connections/edges, the width of edge/line should be based on how strong the correlation is (Spearman’s correlation coefficient), and the color of edge/connecting line should be based on +ve and -ve correlation with other connecting genera. At last, the color of nodes should be the phylum they belongs and there should be a legend penal of colors representing phylum. (Example is attached)

To start plotting the network, literature told me that I should have two files from the data (i.e. Edges file and nodes file). I am really confused how to prepare these two data files from the data I have. Some one please help me how to prepare data, and what should I do in igraph and hmisc packages and when to shift things in Gephi. Hope I asked my question in proper detail. 
Regards

<img width="769" alt="Screenshot 2020-05-29 at 5 44 49 PM" src="https://user-images.githubusercontent.com/66123147/83261430-bc268080-a1d4-11ea-9029-c31632756228.png">

**Comment by eduramiba** [comment]
Hi,
I recommend you follow this tutorial http://www.martingrandjean.ch/gephi-introduction/
